pkg/asset/quota/quota.go

```diff
@@ -135,10 +155,16 @@ func (a *PlatformQuotaCheck) Name() string {
 func summarizeFailingReport(reports []quota.ConstraintReport) error {
 	var notavailable []string
 	var unknown []string
+	var regionMessage string
 	for _, report := range reports {
 		switch report.Result {
 		case quota.NotAvailable:
-			notavailable = append(notavailable, fmt.Sprintf("%s is not available in %s because %s", report.For.Name, report.For.Region, report.Message))
+			if report.For.Region != "" {
+				regionMessage = "in " + report.For.Region
+			} else {
+				regionMessage = ""
+			}
+			notavailable = append(notavailable, fmt.Sprintf("%s is not available %s because %s", report.For.Name, regionMessage, report.Message))
```

Comment:
If there is no region, we end up with 2 spaces between "available" and "because"
```suggestion
			if report.For.Region != "" {
				regionMessage = " in " + report.For.Region
			} else {
				regionMessage = ""
			}
			notavailable = append(notavailable, fmt.Sprintf("%s is not available%s because %s", report.For.Name, regionMessage, report.Message))
```